mmv1/products/bigqueryanalyticshub/terraform.yaml

```diff
@@ -25,4 +25,20 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           data_exchange_id: "my_data_exchange"
           description: "example data exchange"
+  Listing: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: projects/{{project}}/locations/{{location}}/dataExchanges/{{data_exchange_id}}/listings/{{listing_id}}
+    import_format: ["projects/{{project}}/locations/{{location}}/dataExchanges/{{data_exchange_id}}/listings/{{listing_id}}", "{{listing_id}}"]
```

Comment:
Does import with only `{{listing_id}}` actually work? Since location & data_exchange_id can't be pulled from provider config I would guess they cannot work

Reply:
for IAM it doesnt pass without it,remove from resource